modal/_container_entrypoint.py

```diff
@@ -499,10 +499,10 @@ def import_function(function_def: api_pb2.Function, ser_cls, ser_fun, ser_params
 
     # The decorator is typically in global scope, but may have been applied independently
     active_stub = None
-    if isinstance(fun, FunctionHandle):
+    if isinstance(fun, Function):
         _function_proxy = synchronizer._translate_in(fun)
-        fun = _function_proxy.get_raw_f()
-        active_stub = _function_proxy._stub
+        fun = _function_proxy._handle.get_raw_f()
```

Comment:
Some users use `get_raw_f()` directly right now (for example to run local unit tests). Do we need to expose it directly on the proxy?